SW/main/burner_on.py

```diff
@@ -0,0 +1,78 @@
+import Stove
+import numpy as np
+
+class BurnerOn:
+    def __init__(self):
+        self.i = 0
+        self.decreasing = []
+        self.off = True
+        
+    def decreasing_temp(self, average):
+        while self.i < 360:
+            if (self.i % 30 == 0):
+                self.decreasing.append(average)
+            self.i += 1
+            break
+
+        if len(self.decreasing) == 12:
+            dx = np.diff(self.decreasing)
+            self.i = 0
+            self.decreasing = []
+            self.off = np.all(dx <= 0)
+            return np.all(dx <= 0)
+        else:
+            return None
+
+    def update(self, stove):
+        if (stove.upper_right.temp != 0):
+            self.decreasing_temp(stove.upper_right.temp)
+            upper_right_on = self.off
+            if stove.upper_right.pot_detected == False:
```

Comment:
i don't understand the purpose of this code.

Reply:
(checking if pot is detected here)

Reply:
Because we want to only check if burner is left on when pot is not on. Why not check it here?